### ISSUE TITLE
Could you please share the label file of DFDC training set

### ISSUE THREAD

**Issue by Weskiey** [ahaliassos/RealForensics#10]
Hi，I can't find the label file of the training set for the DFDC, could you please share a copy, thank you!

**Comment by ly-code-source** [comment]
Hello, have you obtained the DFDC test set? Can you send me a copy? My email: ly13934301513@163.com
